src/APICore.ts

```diff
@@ -39,7 +39,7 @@ export default class API {
             return await this.requestFile(url, args);
         } catch (error) {
             if (error.name === 'AbortError') {
-                return; // We don't want to resolve or reject the promise
+                return undefined as unknown as Blob;
```

Comment:
This is very shady but we don't want to change the function return type as it might affect every dependant packages

Reply:
I'd say change the return type and return null.

Reply:
that would be a breaking change because it would affect the return type of publicly exposed methods from a non-void to a possible void return type